### PR TITLE
Reintroducing multiQR functionality

### PR DESCRIPTION
**Pull request by brifemu**
In this PR, I have solved the problems occurred from #1699 

- Dependence moment.js was replaced by day.js to solve issues using Safari.
- Code refactored (+100 lines optimized).
- Now generate a PDF, not pages like images.
- The PDF generated can be seen at the site like a slider.
- Working in all browsers.

### CODE REVIEW

File: src/data/eventregistration_de.json [hunk 1]
```diff
@@ -50,7 +50,7 @@
                 "wrongfileheaders": "Die Header der .csv-Datei entsprechen nicht den Headern der entsprechenden Vorlage, Sie können sie <a href='. /event-qr-code-guide/'>hier überprüfen.</a>",
                 "wrongfileformatfields": "Einige Felder in Ihrer CSV-Datei sind nicht korrekt ausgefüllt. Bitte überprüfen Sie Ihre .csv und versuchen Sie es erneut",
                 "filewithnodata": "Ihre .csv Felder sind nicht ausgefüllt",
-                "fileoverloaded": "Die Erstellung von QR-Codes wurde auf 100 . begrenzt"
+                "fileoverloaded": "Die Erstellung von QR-Codes wurde auf 100 begrenzt."
```
Comment (Ein-Tim) [inline src/data/eventregistration_de.json:53]: Wouldn't it make more sense to write "Die Erstellung von QR-Codes ist auf 100 begrenzt." "wurde begrenzt" means that it was possible for some time to create more than 100 QR codes but now it was limited to 100. This is not the case here. 
Same in English.

**Comment by MikeMcC399** [comment]
@dsarkar 
When I execute `npm install` on this branch it makes a change to `package-lock.json`.

image

so it seems that the `package-lock.json` in this branch is not quite in sync with the `package.json` from the branch.

**Comment by dsarkar** [comment]
@MikeMcC399 Yes, it changes package-lock.json, since there are new dependencies.

**Comment by MikeMcC399** [comment]
@dsarkar 
I understand that there are new packages defined in `package.json`
```
"dayjs": "^1.10.7",
"jspdf": "^2.4.0",
"papaparse": "^5.3.1",
```

which lead to changes in `package-lock.json`. My comment was just about a minor difference, not about most of the changes.

Can I suggest the following?
Execute this in `cuisines/cwa-website`:
```
git switch master
npm ci
```
which deletes and re-installs `node_modules` exactly according to the `master `branch version of `package-lock.json`

then
```
git switch brifemu-eventregistration-refactor
npm install
git diff
```

and I think you will see a small difference in `package-lock.json` which should be committed.

When I carried out these steps I got:
```
$ git diff
diff --git a/package-lock.json b/package-lock.json
index 062a3829..a1b7488d 100644
--- a/package-lock.json
+++ b/package-lock.json
@@ -13488,7 +13488,8 @@
       "dependencies": {
         "ansi-regex": {
           "version": "5.0.0",
-          "resolved": ""
+          "resolved": "https://registry.npmjs.org/ansi-regex/-/ansi-regex-5.0.0.tgz",
+          "integrity": "sha512-bY6fj56OUQ0hU1KjFNDQuJFezqKdrAyFdIevADiqrWHwSlbmBNMHp5ak2f40Pm8JTFyM2mqxkG6ngkHO11f/lg=="
         },
         "ansi-styles": {
           "version": "4.3.0",
```

**Comment by MikeMcC399** [comment]
@dsarkar 
One other minor comment,

if you add `papaparse` from the command line

`npm install papaparse@^5.3.1 --save`

npm will add it to `package.json` in alphabetical order in the dependencies:

All the other dependencies are in alphabetical order in  `package.json` which is helpful for maintenance. It does not affect the functionality however.

File: src/data/eventregistration_de.json (hunk 1 above)
Comment (dsarkar) [reply src/data/eventregistration_de.json:53]: fixed in this commit https://github.com/corona-warn-app/cwa-website/pull/2105/commits/c56895e8cbcccada09a59da130e0423820a908d0

Comment (dsarkar) [reply src/data/eventregistration_de.json:53]: alright, you are right, it is about the tense.

Comment (Ein-Tim) [reply src/data/eventregistration_de.json]: Thank you @dsarkar!

Comment (dsarkar) [reply src/data/eventregistration_de.json]: thanks for review

**Comment by svengabr** [comment]
@MikeMcC399 thanks for your input! I did execute your suggested changes.

File: src/data/eventregistration_de.json (hunk 1 above)
Comment (Ein-Tim) [reply src/data/eventregistration_de.json]: You're very welcome!

**Comment by MikeMcC399** [comment]
@svengabr 
> I did execute your suggested changes.

Perfect 👍🏻 

That resolves the issues I pointed out. Thank you!

**Comment by benbender** [comment]
Is there any ETA when this PR will land?
I'm asking especially as there might be a growing demand for this functionality as the constraint to deploy the luca-app may drop in several federal countries...

**Comment by dsarkar** [comment]
@benbender We will have a final look at this, then we should merge it.

**Pull request merged**